### PR TITLE
PWGHF: add max pt for downsampling of candidates to be stored in derived table

### PR DESCRIPTION
**Pull request by fgrosa**

### CODE REVIEW

File: PWGHF/TableProducer/treeCreatorDplusToPiKPi.cxx [hunk 1]
```diff
@@ -210,6 +210,7 @@ struct HfTreeCreatorDplusToPiKPi {
   Configurable<bool> fillOnlySignal{"fillOnlySignal", false, "Flag to fill derived tables with signal for ML trainings"};
   Configurable<bool> fillOnlyBackground{"fillOnlyBackground", false, "Flag to fill derived tables with background for ML trainings"};
   Configurable<float> donwSampleBkgFactor{"donwSampleBkgFactor", 1., "Fraction of background candidates to keep for ML trainings"};
+  Configurable<float> maxPtforDownSample{"maxPtforDownSample", 10., "Maximum pt for the application of the downasampling factor"};
```
Comment (vkucera) [inline PWGHF/TableProducer/treeCreatorDplusToPiKPi.cxx:213]:
```suggestion
  Configurable<float> ptMaxForDownSample{"ptMaxForDownSample", 10., "Maximum pt for the application of the downsampling factor"};
```

File: PWGHF/TableProducer/treeCreatorDplusToPiKPi.cxx [hunk 2]
```diff
@@ -210,6 +210,7 @@ struct HfTreeCreatorDplusToPiKPi {
   Configurable<bool> fillOnlySignal{"fillOnlySignal", false, "Flag to fill derived tables with signal for ML trainings"};
   Configurable<bool> fillOnlyBackground{"fillOnlyBackground", false, "Flag to fill derived tables with background for ML trainings"};
   Configurable<float> donwSampleBkgFactor{"donwSampleBkgFactor", 1., "Fraction of background candidates to keep for ML trainings"};
```
Comment (vkucera) [inline PWGHF/TableProducer/treeCreatorDplusToPiKPi.cxx:212]:
```suggestion
  Configurable<bool> fillOnlyBackground{"fillOnlyBackground", false, "Flag to fill derived tables with background for ML trainings"};
  Configurable<float> downSampleBkgFactor{"downSampleBkgFactor", 1., "Fraction of background candidates to keep for ML trainings"};
```

**Review by vkucera — CHANGES_REQUESTED** [review]

File: PWGHF/TableProducer/treeCreatorDplusToPiKPi.cxx (hunk 2 above)
Comment (vkucera) [reply PWGHF/TableProducer/treeCreatorDplusToPiKPi.cxx:212]: It seems like you fixed the typo but then reverted it.

Comment (fgrosa) [reply PWGHF/TableProducer/treeCreatorDplusToPiKPi.cxx]: Sorry, that was unintentional. I wanted to fix the fact that you changed here the name of the variable, but you kept the old one in the process causing a compilation error. It is now fixed once for all :)

**Review by vkucera — APPROVED** [review]

**Pull request merged**